src/dmd/backend/go.d

```diff
@@ -3,56 +3,129 @@
  * $(LINK2 http://www.dlang.org, D programming language).
  *
  * Copyright:   Copyright (C) 1986-1998 by Symantec
- *              Copyright (C) 2000-2018 by The D Language Foundation, All Rights Reserved
+ *              Copyright (c) 2000-2017 by Digital Mars, All Rights Reserved
```

Comment:
Is this change on purpose? Looks like an accidentally copied header from another file.